modules/tinymce/src/themes/silver/main/ts/ui/window/SilverDialog.ts

```diff
@@ -69,12 +70,26 @@ const renderDialog = <T extends Dialog.DialogData>(dialogInit: DialogManager.Dia
       return Composing.getCurrent(outerForm).getOr(outerForm);
     };
 
+    const toggleFullscreen = (): void => {
+      const fullscreenClass = 'tox-dialog--fullscreen';
+      const sugarBody = SugarElement.fromDom(dialog.element.dom);
+
+      if (!Classes.hasAll(sugarBody, [ fullscreenClass ])) {
```

Comment:
Maybe it's better to use the `Class` module when there is just a single class no extra array needed.

Reply:
done